bucket/mp3directcut.json

```
@@ -0,0 +1,33 @@
+{
+    "##": "Rename to #/dl.7z cannot be used as download is not stripping rename and fosshub cannot handle it",
+    "version": "2.25",
+    "description": "Lossless audio editor and recorder for MP3 files.",
+    "homepage": "http://mpesch3.de",
+    "license": "Freeware",
+    "url": "https://www.fosshub.com/mp3DirectCut.html/mp3DC225.exe",
+    "hash": "md5:872f3923af1982fc39a18d724ffd5904",
+    "installer": {
+        "script": [
+            "Expand-7zipArchive \"$dir\\$fname\" -Removal",
+            "if (-not (Test-Path \"$persist_dir\\mp3DirectCut.ini\")) {",
+            "    Set-Content \"$dir\\mp3DirectCut.ini\" '[mp3DirectCut]' | Out-Null",
+            "}"
```

```suggestion
            "    Set-Content \"$dir\\mp3DirectCut.ini\" @('[mp3DirectCut]', \"version=$version\") | Out-Null",
            "} else {",
            "    (Get-Content \"$persist_dir\\mp3DirectCut.ini\") -replace 'version=.+', \"version=$version\" | Set-Content \"$persist_dir\\mp3DirectCut.ini\"",
            "}",
            "New-Item \"$dir\\mp3DClocal.ini\" | Out-Null"
```
Set version to disable welcome window. mp3DClocal.ini is not really needed. It seems that without it portable mode is still enabled.